### DISCUSSION TITLE
Accuracy of Higher order IFCs

### DISCUSSION THREAD

**Discussion by apandit2021** [ttadano/alamode#26] [Q&A] [answered]
Dear Dr. Tadano,

Hi, how do we test the accuracy of the higher-order (cubic or higher) IFCs calculated by fitting with the harmonic IFCs? I guess we do it by checking the fitting error in the output file but I am not fully sure.

Best,
Abhiyan Pandit

**Comment by ttadano** [comment, marked as answer]
Hi,

> the higher-order (cubic or higher) IFCs calculated by fitting with the harmonic IFCs

The accuracy of higher-order force constants cannot be checked with the harmonic model.
If you want to assess the accuracy of anharmonic force constants, please include the anharmonic terms in the model.

The accuracy of the anharmonic force constants can be checked by increasing the size of the training dataset. If the physical properties, such as thermal conductivity and free energy, change significantly with the size of the training dataset, it indicates that the training is insufficient. If the change is minor, your force constants may be OK.
